src/interfaces/state-interface.ts

```diff
@@ -27,6 +27,7 @@ export interface StateIF {
   lastAnnualReportDate: string // YYYY-MM-DD
   lastAddressChangeDate: string // YYYY-MM-DD
   lastDirectorChangeDate: string // YYYY-MM-DD
+  isHistorical: boolean
```

Comment:
This is probably temporary ... until we get this some other way. I added it here so I could fiddle with it in the unit tests.

Reply:
This will be in a property called state. The value will be 'ACTIVE' or 'HISTORICAL' . It will also have a reason property that indicates what changed this entity to historical state.You will also get the filing id that caused this. This is still pending approval from Thor.

Reply:
Thanks for looking at this with the API in mind. This should be easy to refactor as needed.